sample-apps/java-basic/src/main/java/example/HandlerStream.java

```diff
@@ -1,53 +1,30 @@
 package example;
 
 import com.amazonaws.services.lambda.runtime.Context;
-import com.amazonaws.services.lambda.runtime.LambdaLogger;
 import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
 
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
-import com.google.gson.JsonSyntaxException;
-
-import java.util.HashMap;
 import java.io.InputStream;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.io.InputStreamReader;
-import java.io.BufferedReader;
-import java.io.OutputStreamWriter;
-import java.io.PrintWriter;
-import java.io.BufferedWriter;
-import java.nio.charset.Charset;
-import java.lang.IllegalStateException;
 
 // Handler value: example.HandlerStream
 public class HandlerStream implements RequestStreamHandler {
-  Gson gson = new GsonBuilder().setPrettyPrinting().create();
+
   @Override
   public void handleRequest(InputStream inputStream, OutputStream outputStream, Context context) throws IOException
```

Comment:
You typically use the `RequestStreamHandler` interface when you want to serialise an object in a custom way. I think this would make for a better example.

Reply:
Did you mean this would make a better blank-java example? I think we can leave current the organization as is.